Makefile

```
@@ -25,6 +25,7 @@ verbose ?=      ## Run specs in verbose mode
 junit_output ?= ## Path to output junit results
 static ?=       ## Enable static linking
 interpreter ?=  ## Enable interpreter feature
+check ?=        ## Enable only check when running format
```

I'd use some more explicit name here, like: 

```suggestion
formatter_check ?=        ## Enable only check when running format
```

---

Na, that's just redundant. This flag only has a meaning for the `format` recipe.